samples/browser/src/main/java/org/mozilla/samples/browser/SampleApplication.kt

```diff
@@ -39,6 +41,8 @@ class SampleApplication : Application() {
         components.engine.warmUp()
 
         try {
+            GlobalAddonManagerProvider.initialize(components.addonManager)
+            GlobalAddonUpdaterProvider.initialize(components.addonUpdater)
```

Comment:
I know we need one of those for the worker, but don't think we need two? I think we should just pass the manager to the updater's constructor. The updater is the only one accessing the 'GlobalAddonManagerProvider' anyway. So, we can make that dependency explicit.

Reply:
Updated